sdks/python/setup.py

```diff
@@ -95,16 +95,6 @@ def get_version():
 different technologies and user communities.
 '''
 
-REQUIRED_PIP_VERSION = '7.0.0'
```

Comment:
The context was here: https://github.com/apache/beam/issues/22218#issuecomment-1196014115. We may change REQUIRED_PIP_VERSION -> RECOMMENDED_MIN_PIP_VERSION and change the warning accordingly. And put the code in a try clause and catch `DistributionNotFound`.

Reply:
That sounds better and more robust. Done.

Reply:
Thanks!